staging_upload.py

```diff
@@ -17,9 +21,14 @@
 max_concurrent_aiohttp_streams = 10
 # map to copy artifacts to bucket - public/build/target.complete.mar
 
-# where are the mac / win mars?
-# https://archive.mozilla.org/pub/devedition/tinderbox-builds/jamun-macosx64-devedition/1494964280/
 
+LATEST_MAR_LOCATIONS = {
+    'win32': 'https://index.taskcluster.net/v1/task/gecko.v2.jamun.latest.devedition.win32-opt/artifacts/public/build/firefox-54.0.en-US.win32.complete.mar',
+    'win64': 'https://index.taskcluster.net/v1/task/gecko.v2.jamun.latest.devedition.win64-opt/artifacts/public/build/firefox-54.0.en-US.win64.complete.mar',
```

Comment:
I'd love to see these templatized, so someone could specify 'jamun' or 'mozilla-beta', and the version number, and get the expected mar locations.  (My intent was to take the hardcoded vars at the top, and turn them into commandline options via argparse at some point.)